atom/browser/web_contents_preferences.cc

```diff
@@ -129,51 +199,58 @@ WebContentsPreferences* WebContentsPreferences::From(
 
 void WebContentsPreferences::AppendCommandLineSwitches(
     base::CommandLine* command_line) {
-  bool b;
   // Check if plugins are enabled.
-  if (dict_.GetBoolean(options::kPlugins, &b) && b)
+  if (IsEnabled(options::kPlugins))
     command_line->AppendSwitch(switches::kEnablePlugins);
 
   // Experimental flags.
-  if (dict_.GetBoolean(options::kExperimentalFeatures, &b) && b)
+  if (IsEnabled(options::kExperimentalFeatures))
     command_line->AppendSwitch(
         ::switches::kEnableExperimentalWebPlatformFeatures);
 
   // Check if we have node integration specified.
-  bool node_integration = true;
-  dict_.GetBoolean(options::kNodeIntegration, &node_integration);
+  bool enable_node_integration = true;
+  auto* node_integration_value = preference_.FindKeyOfType(
```

Comment:
We have this lovely `IsEnabled` helper, can we make that take a default value and return it if no value was found?

Reply:
Oh it already takes a default value, can we just use that helper?